setup.py

```diff
@@ -15,38 +15,17 @@
     classifiers=[
         "Programming Language :: Python",
         "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
+        "Topic :: Office/Business :: Financial :: Investment",
     ],
-    packages=find_packages(exclude=["tests"]),
+    packages=find_packages(exclude=["tests", "sample_data"]),
     namespace_packages=[],
     include_package_data=False,
     zip_safe=False,
-    install_requires=[
-        "pandas",
-        "openpyxl",
-        "pydantic",
-        "PyYAML",
-        "click",
-        "python-dotenv",
-        "ipython",
-        "jupyter",
-        "jupytext",
-        "seaborn",
-        "yfinance",
-        "xlrd",
-    ],
+    install_requires=[line.strip() for line in open("requirements-latest.txt")],
```

Comment:
you can add helper method for reading lines, and preferably use "with open" to close the file faster

Reply:
Yea, that was super lazy. There was even a function reading the README already